package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trayio/falafel",
-  "version": "1.28.0",
+  "version": "1.28.1",
```

Comment:
Given setting ACL is now default behaviour, I think the change warrant a minor bump instead of patch

Reply:
(regen package-lock too)

Reply:
Yea was wondering if this should now be a minor bump. Cool so bumped to 1.29.0